TestAnimals/UnitTest1.cs

```diff
@@ -0,0 +1,45 @@
+using AnimalsTest;
+
+namespace TestAnimals
+{
+    public class UnitTest1
+    {
+        [Fact]
+        public void VoiceLion()
+        {
+            var lion = new Lion();
+            var actual = lion.Voice();
```

Comment:
```suggestion
            string actual = lion.Voice();
```

Reply:
This is not needed really, you can do `Assert.Equal(expectedVoice, lion.Voice());`